src/startup/client.jsx

```diff
@@ -1,6 +1,7 @@
 import { ApolloClient, InMemoryCache } from '@apollo/client'
```

Comment:
Since we're not using any jsx syntax here, should we change this file's extension to `.js`?

Reply:
Done